inc/genhdr/qstrdefs.generated.h

```diff
@@ -382,6 +382,19 @@ QDEF(MP_QSTR_PITCHFORK, (const byte*)"\x33\x09" "PITCHFORK")
 QDEF(MP_QSTR_XMAS, (const byte*)"\xe2\x04" "XMAS")
 QDEF(MP_QSTR_PACMAN, (const byte*)"\x55\x06" "PACMAN")
```

Comment:
Hey @larryhastings it's already been merged in some weeks ago. Surely this applies as "fair use" (educational)..? IANAL. ;-)